apache/superset
# feat: Migrates TreeMap chart

### SUMMARY
This PR migrates the Treemap chart to the ECharts version.

- Removes the legacy Treemap code and unregisters the plugin.
- Removes old plugin references throughout the codebase
- Updates unit tests
- Updates E2E tests
- Updates the example dashboards
- Updates translation files

Note a previous migration existed however the legacy chart wasn't removed and thus new legacy Treemap charts could have been created.

### AFTER SCREENSHOT
<img width="1077" alt="Screenshot 2023-04-19 at 11 48 43" src="https://user-images.githubusercontent.com/70410625/233114191-eb6a6bd4-15f2-4366-8c62-c77336f53f23.png">

### TESTING INSTRUCTIONS
1 - Make sure all Treemap (legacy) charts were converted to Treemap
2 - Make sure Treemap (legacy) is not available anymore in the viz picker
3 - Make sure that it's possible to revert the migration by executing superset db downgrade and reverting this PR

### ADDITIONAL INFORMATION
